docs/configuration_options.md

```diff
@@ -147,7 +147,7 @@ scylla cloud cluster name
 
 **default:** N/A
 
-**type:** str
+**type:** str (appendable)
```

Comment:
how `cloud_cluster_name` is related to this change ?

Reply:
It is not, i included it because not up to date check, the issue was since fixed so I will try rebase